nautobot/utilities/tests/test_paginator.py

```diff
@@ -74,7 +80,9 @@ def test_enforce_max_page_size(self):
             self.assertEqual(len(response.context["table"].page), 10)
         with self.subTest("global config PAGINATE_COUNT=50 returns 10 rows"):
             self.user.clear_config("pagination.per_page", commit=True)
-            response = self.client.get(url)
+            response = self.client.get(reverse("circuits:provider_list"))
```

Comment:
Making sure im also testing on NautobotUIViewSet

Reply:
Maybe add a comment to that effect?